topics/admin/tutorials/data-library/tutorial.md

```diff
@@ -0,0 +1,269 @@
+
+> ### {% icon code-in %} Input: Bash
+> ```bash
+> cat /libraries/example-library.yaml
+> ```
+{: .code-in}
+
+> ### {% icon code-out %} Output: Bash
+> ```yaml
+> ---
+> destination:
+>   type: library
+>   name: Mouse sequencing project
+>   description: some data
+>   synopsis: samples collected from somewhere
+> items:
+> - url: https://zenodo.org/api/files/287966da-5411-4f79-8cfb-0ffa84d0d6cc/wildtype.fna
+>   src: url
+>   ext: fasta
+>   info: https://doi.org/10.5281/zenodo.582600
+> - name: A directory
+>   description: Exome sequencing means that all protein-coding genes in a genome are
+>   items:
+>   - url: https://zenodo.org/api/files/287966da-5411-4f79-8cfb-0ffa84d0d6cc/mutant_R1.fastq
+>     src: url
+>     ext: fastqsanger
+>     info: https://doi.org/10.5281/zenodo.582600
+>   - url: https://zenodo.org/api/files/287966da-5411-4f79-8cfb-0ffa84d0d6cc/mutant_R2.fastq
+>     src: url
+>     ext: fastqsanger
+>     info: https://doi.org/10.5281/zenodo.582600
+> ```
+{: .code-out}
+
+Let's try setting that up in our Galaxy!
+
+{% include snippets/finding_your_admin_api_key.md %}
+
+> ### {% icon hands_on %} hands-on: importing data library yaml.
+>
+> 1. re-activate the virtualenv you created for the [ephemeris tool management tutorial]({% link topics/admin/tutorials/tool-management/tutorial.md %}).
+>
+>    > ### {% icon code-in %} input: bash
+>    > ```
+>    > . ~/ephemeris_venv/bin/activate
+>    > ```
+>    {: .code-in}
+>
+>    > ### {% icon tip %} missing?
+>    >
+>    > then you might need to re-run the steps:
+>    >
+>    > ```bash
+>    > virtualenv -p python3 ~/ephemeris_venv
+>    > . ~/ephemeris_venv/bin/activate
+>    > pip install ephemeris
+>    > ```
+>    {: .tip}
+>
+> 2. we'll use the `setup-data-libraries` command to install the data in this yaml file into a library in our galaxy.
+>
+>    > ### {% icon code-in %} input: bash
+>    > ```
+>    > setup-data-libraries -g https://your-galaxy -k <api-key> --training -i /libraries/example-library.yaml --legacy
+>    > ```
+>    {: .code-in}
+>
+>    > ### {% icon code-out %} output
+>    > ```
+>    > library name: mouse sequencing project
+>    > ```
+>    {: .code-out}
+>
+> 3. this command is safe to re-run. for example if you update the yaml, it will simply report that the library exists before ensuring all files exist in their appropriate location:
+>
+>    ```bash
+>    library name: mouse sequencing project
+>    library already exists! id: f597429621d6eb2b
+>    ```
+>
+{: .hands_on}
+
+That's it! It's quite easy. Note that we've used some special flags here, `--training` and `--legacy`. Training sets some defaults that make sense for the GTN (mostly around library descriptions / etc.)
+
+![Screenshot of data libraries, we're in a library folder named "Mouse sequencing project" and a directory and single file are shown. The file has an ugly URL as its name](../../images/data/imported.png)
```

Comment:
Move the image before or after? It's in the middle of the --legacy explanation currently

Reply:
fixed, I think.